Environments/SharePoint-AllVersions/azuredeploy.json

```diff
@@ -0,0 +1,1210 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "provisionSharePoint2013": {
```

Comment:
Would it make sense that these are ordered to show the latest on top (i.e. 2019, 2016, then 2013)? And only default to enabling 2019?

Reply:
I went ahead and made the change. Do bring it up if there's a more common scenario that we should consider.